openmcyclus/depletion.py

```diff
@@ -165,16 +162,15 @@ def get_spent_comps(self, material_ids, path):
             list of the compositions from the OpenMC model
         '''
         results = od.Results(path + "depletion_results.h5")
+        nuclides = od.MicroXS.from_csv(path+"micro_xs.csv").nuclides
```

Comment:
Why load in the `MicroXS` object again? Why not just use `self.micro_xs.nuclides`?

Reply:
Because I don't actually initialize that object with the class. That might be a cleaner way to do this though. I don't have strong feelings either way. I think initially I was thinking that you wouldn't always need that, but it also might be good to initialize the materials with the class as well.

Reply:
I'm giving this a shot (initializing the cross sections and materials with the class) see the changes in commit `c1d7d2` and let me know what you think.